.github/scripts/install.sh

```diff
@@ -56,6 +56,9 @@ popd
 # To check our manifest
 pip install check-manifest
 
+# TODO: update setup.py when package is avalable in defaults
```

Comment:
For alphas and betas the idea is to add new packages to our channel and install them from there (else the first conda package we create for 5.0 wouldn't have the right dependencies).

So please do that and change the installation script and dependencies files below accordingly.

Reply:
If I try to add the `spyder`channel to the installation of `conda.txt` it will not work and enter in conflicts. So the spyder channel needs to be emptyfrom anything other than the extra dependencies not found in defaults.

Reply:
Ok fixed

Reply:
Please remove this comment and the line below then.